packages/nominal-types/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2018 Solana Labs, Inc
```

Comment:
I'm sure this is copy-pasted from somewhere in Kit, but are we meant to be using our own copyright for Kit stuff?

Reply:
Yeah, let's fix these all at once. I'll do this one now ofc.